src/RandomArt/RandomArt.js

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from 'react'
+import { Link } from 'react-router-dom'
+import usePaintings from '../Hooks/usePaintings';
+import Gallery from '../Gallery/Gallery';
+import backBtn from '../assets/back-btn.png'
+import tagBtn from '../assets/tagIcon.png'
+import randomTerms from './randomTerms'
+
+function RandomArt(props) {
+
+  const [reload, setReload] = useState(false)
+  
+  const randomWord = randomTerms[Math.floor(Math.random() * randomTerms.length)]
+  const randomPaintings = usePaintings(`http://www.wikiart.org/en/search/${randomWord}/1?json=2`)
+  const newWord = randomTerms[Math.floor(Math.random() * randomTerms.length)]
+  const newRandomPaintings = usePaintings(`http://www.wikiart.org/en/search/${newWord}/1?json=2`)
+
+
+  return (
+    <section className="painter-page">
+       <section className="painter-nav">
+          <Link to={"/"} style={{ textDecoration: 'none' }}>
+            <img src={backBtn} alt='back-btn' className='back-btn' />
+          </Link>
+          <button className="random-art-btn" onClick={() => {
+                                                if(reload === true) {
+                                                  setReload(false)
+                                                }
+                                                if(reload === false){
+                                                  setReload(true)
+                                                }      
+                                              }
+          }>
+              More Art
+          </button>
+          <img src={tagBtn} alt='save-btn' className='save-btn' />
+      </section>
+      <section aria-label="gallery">
+        {reload === true ? (<Gallery paintings={randomPaintings} setSelected={props.setSelected}/>) : (<Gallery paintings={newRandomPaintings} setSelected={props.setSelected}/>) }  
```

Comment:
I like where you're going with this idea, but the logic feels a little hairy. What happens if the user wants to reload more random art? Do they click [this button](https://github.com/carflor/ArtisTry/pull/36/files#diff-199d58ac76e4af5a84949e2341a71325R25) or the [button from the App component](https://github.com/carflor/ArtisTry/pull/36/files#diff-1d8649bbc6e0907ffd551dce8dde6bafR22)? Or does the latter button only get rendered when we're at the `/` route path?

If you can only randomly load 100 random painting without having to re-mount this component, what's a way that you could make it so that a user can indefinitely get new random paintings from this component?

I also may be misunderstanding the logic here -- feel free to say if I'm off base!